### DISCUSSION TITLE
How can I open the 2nd macro window in the Klipper screen?

### DISCUSSION THREAD

**Discussion by bybarbaros** [KlipperScreen/KlipperScreen#1212] [General]
hello . I need your information and guidance on two issues.

1.) I have the LCD panel you see in the picture (picture 1) to be used in my system. How can I adjust the contrast and brightness of this panel?...

2.) My second question: I created a picture (picture 2) to explain what I want to do. I want to create a second macro page and control some of the macros I will add on the Klipper screen from within this page. Is it possible to create such a second "makro2" page? how can I do it. Thank you in advance for your ideas, suggestions and help.!

![picture_2.png](https://github.com/KlipperScreen/KlipperScreen/assets/118777939/15faa78f-b281-4b54-ab44-a07732c28359)

![picture_1.jpg](https://github.com/KlipperScreen/KlipperScreen/assets/118777939/906ec638-3cd9-441a-a1e9-7c82044cf605)

**Comment by alfrix** [comment]
to have a selection of macros on a submenu you need to create the menu and all the the submenu entries, described here: https://klipperscreen.readthedocs.io/en/latest/Configuration/#menu

ks doesn't support changing brightness, since there is no standarized way to do it and most of the raspberry screens don't support changing it anyways

**Reply by bybarbaros** [reply]
thanks a lot. I will start my experiments as soon as possible.
